pitr/cli/internal/cmd/backup.go

```diff
@@ -253,6 +257,37 @@ func execBackup(lsBackup *model.LsBackup) error {
 	return nil
 }
 
+func checkAgentServerStatus(lsBackup *model.LsBackup) bool {
+
+	type Status struct {
```

Comment:
I think move struct definition out of func is better

Reply:
done